hippo_test.go

```diff
@@ -139,6 +139,7 @@ func TestSinglePartBatch_Error(t *testing.T) {
 
 		w.Header().Set("Content-Type", "application/json")
 		w.WriteHeader(200)
+		// nolint
 		w.Write(responseBytes)
```

Comment:
curious - why not modify the line to `_, _ = w.Write(responseBytes` (just like line 68 of this same file) instead of adding the `// nolint` line?

Reply:
I had it this way but decided that nolint is more straightforward..
And I thought I have changed both places..

So what way is better? I dont have clear winner.. (just all places should keep same convention)

Reply:
I like to use the `_` when intentionally ignoring errors. I have never used `//nolint` personally.
Is the linter configured to fail the pipeline on this error?  In my experience, unused error values are just a warning.

Reply:
Yes..

Reply:
Actually its not about unused variables but about unhandled error..

Reply:
My $.02:

I prefer assignment assignment to `_, _` because it explicitly tells code-readers (on Github, w/o IDE) that the code-writer intentionally ignored the return and error values.

On the other hand, the `// nolint` says that the code-writer is ignoring something that the linter disliked, but code-readers would have to guess/assume what that something is.

Reply:
Thanks Guys..
Its `go`ism, but yes its much cleaner and as @aaronfuj mentioned comments can be moved while refactoring..
I rewrote all and also removed all `// nolint` for checking `unit < 0`

So there is one just one nolint for unused value but Im not 100000% sure if its really unused so I left it there..

Reply:
Late to weigh in here, but I agree that I'd rather see `_, _ = ` instead of `// no lint`. 

Unless it's a pain otherwise, it's cleaner to configure the tools, rather than leave comments _for_ the tools in the code.